### PR TITLE
Feature/implement dashboard ui

### PR DESCRIPTION
**Pull request by RobertWSaunders**
### Changes proposed in this PR:
- adds a basic splash to the dashboard ui

### CODE REVIEW

File: client/components/Dashboard/Dashboard.js [hunk 1]
```diff
@@ -18,9 +20,25 @@ class Dashboard extends Component {
...
+						<p>What if I don't get accepted? We'll have a waiting list that you can enter into from the dashboard and it will be a first come first serve basis. We really wish that we could accept everyone but we receive thousands of applications and just do not have the infrastructure to maintain the quality and safety of our event. Every year there are always those applicants that reach out to us asking if there is anything we can do to help them and our advice is to just make sure you are looking for that email and jump on the waiting list as soon as you can, we can't make any exceptions because it wouldn't be fair.</p>
+						<p>While you wait to hear from us there are some things you can do, such as:</p>
+						<ul>
+							<li>Like us on <a href="">Facebook</a></li>
```
Comment (zackharley) [inline client/components/Dashboard/Dashboard.js]: Need to add FB link

File: client/components/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -18,9 +20,25 @@ class Dashboard extends Component {
...
+							<li>Follow us on <a href="">Twitter</a></li>
```
Comment (zackharley) [inline client/components/Dashboard/Dashboard.js]: Need to add Twitter link

File: client/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -18,9 +20,25 @@ class Dashboard extends Component {
...
+							<li>Follow us on <a href="">Instagram</a></li>
```
Comment (zackharley) [inline client/components/Dashboard/Dashboard.js]: Need to add Instagram link

**Comment by RobertWSaunders** [comment]
Good catch! Silly mistakes, thanks 👍

**Pull request merged**